docs/components/tabs.md

```diff
@@ -369,3 +369,96 @@ Token                                       | Default value
   <md-secondary-tab>Tab 3</md-secondary-tab>
 </md-tabs>
 ```
+
+<!-- auto-generated API docs start -->
+
+## API
+
+
+### MdTabs
+
+#### Properties
+
+<!-- mdformat off(autogenerated might break rendering in catalog) -->
+
+Property | Attribute | Type | Default | Description
+--- | --- | --- | --- | ---
+`autoActivate` | `auto-activate` | `boolean` | `false` | Whether or not to automatically select a tab when it is focused.
+`tabs` |  | `Tab[]` | `undefined` | The tabs of this tab bar.
+`activeTab` |  | `Tab` | `undefined` | The currently selected tab, `null` only when there are no tab children.
+`activeTabIndex` |  | `number` | `undefined` | The index of the currently selected tab.
+
+<!-- mdformat on(autogenerated might break rendering in catalog) -->
+
+#### Methods
+
+<!-- mdformat off(autogenerated might break rendering in catalog) -->
+
+Method | Parameters | Returns | Description
+--- | --- | --- | ---
+`scrollToTab` | `tabToScrollTo` | `Promise<void>` | Scrolls the toolbar, if overflowing, to the active tab, or the provided<br>tab.
+
+<!-- mdformat on(autogenerated might break rendering in catalog) -->
+
+#### Events
+
+<!-- mdformat off(autogenerated might break rendering in catalog) -->
+
+Event | Type | Description
+--- | --- | ---
+`change` | `undefined` | Fired when the selected tab changes. The target's selected or<br>selectedItem and previousSelected or previousSelectedItem provide information<br>about the selection change. The change event is fired when a user interaction<br>like a space/enter key or click cause a selection change. The tab selection<br>based on these actions can be cancelled by calling preventDefault on the<br>triggering `keydown` or `click` event.
+
+<!-- mdformat on(autogenerated might break rendering in catalog) -->
+
+### MdPrimaryTab
+
+#### Properties
+
+<!-- mdformat off(autogenerated might break rendering in catalog) -->
+
+Property | Attribute | Type | Default | Description
+--- | --- | --- | --- | ---
+`inlineIcon` | `inline-icon` | `boolean` | `false` | Whether or not the icon renders inline with label or stacked vertically.
+`active` | `active` | `boolean` | `false` | Whether or not the tab is selected.
+`hasIcon` | `has-icon` | `boolean` | `false` | In SSR, set this to true when an icon is present.
+`iconOnly` | `icon-only` | `boolean` | `false` | In SSR, set this to true when there is no label and only an icon.
+`selected` | `selected` | `boolean` | `undefined` | TODO(b/293476210): remove after migrating
+
+<!-- mdformat on(autogenerated might break rendering in catalog) -->
+
+#### Methods
+
+<!-- mdformat off(autogenerated might break rendering in catalog) -->
+
+Method | Parameters | Returns | Description
+--- | --- | --- | ---
+`[ANIMATE_INDICATOR]` | `previousTab` | `void` | 
```

Comment:
This public method uses a symbol to obfuscate it from public use. Should probably be omitted from public docs.